tests/plugins/test_common.py

```diff
@@ -0,0 +1,108 @@
+import os
```

Comment:
super nit: test_common quickly becomes a dumping ground for tests. maybe this can be moved to https://github.com/PyTorchLightning/pytorch-lightning/tree/master/tests/plugins/environments as `test_cluster_environment.py`?

Reply:
yes, I was wondering myself. but hard decision, it is not clear where the tests should go. they are neither purely testing environment nor just about the plugins.

Reply:
what about `test_cluster_integration.py`? IMO that's better. So it could be the same file, just renamed to avoid dumping everything else in here.

Reply:
good suggestion